script/parse_indexing_log.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+#
+# parses a `log/indexer.log` and outputs a CSV file of the timings.
+# Use a dash (-) as an argument to read from STDIN.
+#
+# The data output are the real time as `:real` and percentage of time
+# spend idle as `:pct_idle` (e.g., waiting for network requests), for
+# the time it takes to read the core data from Fedora into an object
+# `load:` and the time it takes to convert that object into a solr
+# document `:to_solr`
+#
+# Usage: script/parse_indexing_log.rb [-] > data.csv
+#
+puts %w(druid load:real load:pct_idle to_solr:real to_solr:pct_idle total:real).join(',')
+
+input = ARGV.shift == '-' ? STDIN : open('log/indexer.log')
+input.each do |line|
+  matches = %r{successfully updated index for druid:([a-z0-9]+).*metrics: load_instance realtime ([\d\.]+)s total CPU ([\d\.]+)s; to_solr realtime ([\d\.]+)s total CPU ([\d\.]+)s}.match(line)
+  if matches
+    record = [
+      matches[1],
```

Comment:
Could we use named captures to clean this up? I'm a little confused which values are what.